spec/aws_support.rb

```diff
@@ -44,7 +44,7 @@ def purge_all
 
   def setup_public_vpc
     aws_vpc 'test_vpc' do
-      cidr_block '10.0.0.0/24'
+      cidr_block '10.0.0.0/16'
```

Comment:
haha, I had to do the same thing in https://github.com/chef/chef-provisioning-aws/pull/346/files#diff-909db8c1e9f02815c26e8ebd3caa7b49R47